tests/test_connector_matrix.py

```diff
@@ -460,7 +460,12 @@ async def test_parse_sync_response(self, connector):
                 nio.ProfileGetDisplayNameResponse("SomeUsersName")
             )
 
-            returned_message = await connector._parse_sync_response(self.sync_return)
+            returned_message = [
+                m async for m in connector._parse_sync_response(self.sync_return)
```

Comment:
I don't think it matters too much in this context but this isn't especially easy to read, for me at least.

Reply:
And I don't know what you would do about that anyway. Just an observation really.